tools/ai2_common.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
+# 0-*- Mode: Python; tab-width: 4; indent-tabs-mode: nil; coding: utf-8; -*-
+
+import sys
+import os.path
+import spacy
+import itertools
+import glob
+import re
+from sys import path as sys_path
+
+# this seems to be necessary for annotations to find its config
+sys_path.append(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Sorry, this is just some voodoo I copied from the other python scripts in the directory. I haven't yet investigated why it is as it is.